ruby/vampires.rb

```diff
@@ -0,0 +1,106 @@
+#Now we ask how many applicants need to be interviewed!
+i = 0
+n = 0
+
+while i == 0
+	puts "How many employees will you be interviewing?"
+	i = gets.chomp.to_i
+	n += 1
+	if n == 3
+	  puts "Clearly a cat is typing on the keyboard, adios furball!"
+	  i += 1
+	elsif i == 0
+		puts "That is not a number, please re-enter. (If typed out as word, please enter as integer.)"
+	end
+end
+
+while (i > 0) && (n < 3)
+	i -= 1
+
+	#This is the questionaire section!
+	puts "What is your name?"
+	name = gets.chomp.downcase
+
+	age = 0
+	while age == 0
+		puts "How old are you? (please use a number)"
+		age = gets.chomp.to_i
+		if age == 0
+			puts "Error, that is not a number. If you want this job you really need to work on your reading comprehension! (Or you spelled it out, please use numberpad)"
+		end
+	end
+
+	year = 0
+	while year == 0
+		puts "What year were you born?(please use a four-digit number)"
+		year = gets.chomp.to_i
+		if year == 0
+			puts "Error, that is not a year. Are you even taking this job interview seriously? (Please use the numberpad!)"
+		end
+	end
+
+	true_age = 2017 - year
+
+	garlic = 0
+	while (garlic != "y") && (garlic != "n")
+		puts "Our company cafeteria serves garlic bread. Should we order some for you? (y/n)"
+		garlic = gets.chomp
+		if (garlic != "y") && (garlic != "n")
+			puts "That neither a 'y or a 'n'. I really need to have a discussion with our recruiter."
+		end
+	end
+
+	insurance = 0
+	while (insurance != "y") && (insurance != "n")
+		puts "Would you like to enroll in the company's health insurance?(y/n)"
+		insurance = gets.chomp
+		if (insurance != "y") && (insurance != "n")
+			puts "That's neither a...you know what, I know you're just messing with me now."
+		end
+	end
+
+	# This is the vampireness-calculation section!
+
+	vampire = 0
+	if ((age == true_age) || (age == true_age - 1)) && ((garlic == "y") && (insurance == "y"))

```

Comment:
seems like there is a bit of repetition between checking if `garlic == 'y'` or if age is correct in some manner.

Look into something such as `likes_garlic = garlic == 'y'` then we can use that as our boolean value; if we need to see if they don't like garlic we can do `!likes_garlic`